### ISSUE TITLE
[PRCI - demo] change wording "Request Summary" to "Request Submitted" on confirmation page after submitting a request

### ISSUE THREAD

**Issue by Clementinev** [blindnet-io/privacy-components-web#214]
On the demo, on PRCI, on the confirmation page after submitting a request, the wording "Request Summary" is not indicative enough to understand this page is the confirmation that the request has been submitted (the change of highlight tab on top of page is not enough). 
What do you think of changing the wording "Request Summary" to "Request Submitted" ? 
<img width="1440" alt="Request submitted" src="https://user-images.githubusercontent.com/89908145/201122983-8ee87c60-68f6-4546-8a35-357d40bc0b27.png">

**Comment by Clementinev** [comment]
Now I understand it is the same page as the request details page from request history, I think "Request Submitted" still works for both, let me know what you think

**Comment by jboileau99** [comment]
@Clementinev  I'm not sure, I imagine this box as the container of the whole request --> the request summary, and then I think the "Under Review" gives confirmation that the request has been submitted, do you agree?

**Comment by Clementinev** [comment]
I agree it makes sense to keep "Request Summary", I think we should still think of some stronger visual feedback (something that changes/stand out from previous screen to clearly indicate request is submitted) for example a text line under "back to request" button saying Your request has been submitted! with a green tick/check icon, something more explicit that kind of "reassure" the user as well
